client/repo_pkcs11.go

```diff
@@ -25,9 +25,19 @@ func NewNotaryRepository(baseDir, gun, baseURL string, rt http.RoundTripper,
 	}
 
 	keyStores := []trustmanager.KeyStore{fileKeyStore}
+	if UseNative {
+		nativeKeyStore, err := trustmanager.NewKeyNativeStore(passphrase.ConstantRetriever("password"))
```

Comment:
same comment here as above with respect to using a prompt retriever instead of a constant `"password"`